src/components/AnonymousChat/MessageChannelItem.tsx

```diff
@@ -77,6 +78,9 @@ const MessageChannelItem: (props: MessageChannelItemProps) => React.ReactElement
       channelType={item?.channelType}
       dbAnonUserInfo={dbAnonUserInfo}
       hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
+      containerBackgroundColor={
+        item?.channelType === 'PM' ? COLORS.signed_secondary : COLORS.anon_secondary
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting the `containerBackgroundColor` is straightforward and readable. However, it's not very scalable. If more channel types are added in the future, this ternary operation would become complex and hard to maintain. Consider using a mapping object or a function to handle this.

Here's an example of how you could refactor this:

```diff
- containerBackgroundColor={
-   item?.channelType === 'PM' ? COLORS.signed_secondary : COLORS.anon_secondary
- }

+ const getBackgroundColor = (channelType) => {
+   switch(channelType) {
+     case 'PM':
+       return COLORS.signed_secondary;
+     default:
+       return COLORS.anon_secondary;
+   }
+ }
+
+ containerBackgroundColor={getBackgroundColor(item?.channelType)}
```
This way, if new channel types are introduced, you can simply add a new case to the `getBackgroundColor` function.


<!-- This is an auto-generated comment by OSS CodeRabbit -->